### PR TITLE
fix 958 classif.glmnet in multilabelBinaryRelevanceWrapper

### PR DESCRIPTION
**Pull request by Coorsaa**
fixed bug from issue #958.
set subset to NULL in predict function in order to prevent the parameter s of glmnet from getting partial matching problems.

### CODE REVIEW

**Comment by larskotthoff** [comment]
Could you also add a unit test for this please?

**Comment by berndbischl** [comment]
*at least* the thing from the issue report

File: R/MultilabelBinaryRelevanceWrapper.R [hunk 1]
```diff
@@ -50,11 +50,11 @@ trainLearner.MultilabelBinaryRelevanceWrapper = function(.learner, .task, .subse
 }
 
 #' @export
-predictLearner.MultilabelBinaryRelevanceWrapper = function(.learner, .model, .newdata, ...) {
+predictLearner.MultilabelBinaryRelevanceWrapper = function(.learner, .model, .newdata, .subset = NULL, ...) {
```
Comment (giuseppec) [inline R/MultilabelBinaryRelevanceWrapper.R]: We don't use `.subset` in the function, why?

**Comment by larskotthoff** [comment]
Is this ready to be merged? @Coorsaa  @giuseppec

**Comment by Coorsaa** [comment]
@larskotthoff: I think so, however, @giuseppec might want to look into it again.

**Comment by giuseppec** [comment]
Looks good to me.

**Comment by larskotthoff** [comment]
Ok, thanks. Just one question -- in `predict.R` you're changing the default for `subset` to `NULL`. Why?

**Comment by Coorsaa** [comment]
glmnet has a parameter "s" so we had a partial matching with the subset if the default was not set as discussed in issue #958

**Comment by larskotthoff** [comment]
Oh yes and you actually said that before. Sorry, I missed that!

**Pull request merged**